src/main.ts

```diff
@@ -117,14 +120,8 @@ export default class TTSPlugin extends Plugin {
 						}));
 				});
 			} else {
-				menu.addItem((item) => {
-					item
-						.setIcon("play-audio-glyph")
-						.setTitle("Play")
-						.onClick((async () => {
-							await this.ttsService.play(markdownView);
-						}));
-				});
+				await this.ttsService.play(markdownView);
```

Comment:
It was inconvinient to click to buttons to start the playback. So now if there is no active playback start a new one without creating the menu.